src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -91,7 +91,10 @@ export default Object.freeze({
   mhvSecureMessagingToVaGovRelease: 'mhv_secure_messaging_to_va_gov_release',
   mhvToLogingovAccountTransition: 'mhv_to_logingov_account_transition',
   mhvToLogingovAccountTransitionModal: 'mhv_to_logingov_account_transition_modal',
+  myVaEnableNotificationComponent: 'my_va_notification_component',
   myVaUseExperimental: 'my_va_experimental',
+  myVaUseExperimentalFrontend: 'my_va_experimental_fe',
+  myVaUseExperimentalFullstack: 'my_va_experimental_fs',
```

Comment:
I'm wondering if these should be:

```  
myVaUseExperimentalFe: 'my_va_experimental_fe',
myVaUseExperimentalFs: 'my_va_experimental_fs',
```

Instead of using the full word, or maybe using the full word: `my_va_experimental_frontend` for the camel cased names as well? May make the cognitive load of connecting the two easier if they were exactly the same and just camelCase vs snake_case.

Reply:
But I'm also seeing things like: ` manageDependents: 'dependents_management',` in the file already so 🤷🏼